### PR TITLE
Timeline

### PR DESCRIPTION
**Pull request by eloiseespel**

### CODE REVIEW

File: Timeline.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Timeline for high school 
+October 27th � When we want our new requirements document to be done
```
Comment (Ansper11) [inline Timeline.txt:2]: I agree, then we can have enough time to code

File: Timeline.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+October 28th � Start coding prototype 
+November 4th � Requirements are due 
+Milestone 1 � Coded prototype with basic features any AI or ML is mocked 
+Milestone 2 � Basics AI or ML using libraries + lesson is coded and functional 
+Final milestone � Polishing, AI/ML is finishes, all testing and bugs have been taken care of
```
Comment (yyht6300a) [inline Timeline.txt:7]: It's a clear timeline for us.

File: Timeline.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Timeline for high school 
+October 27th � When we want our new requirements document to be done
+October 28th � Start coding prototype
```
Comment (Nick-McGee) [inline Timeline.txt:3]: This is a good time to start the prototype.

File: Timeline.txt (hunk 1 above)
Comment (eloiseespel) [reply Timeline.txt:2]: 2 weeks to code would definitely be too short, I'm glad the team agrees on starting early

**Pull request merged**